src/FunctionApps/python/IntakePipeline/linkage.py

```diff
@@ -0,0 +1,2 @@
+def add_patient_identifier(bundle: dict) -> None:
+    pass
```

Comment:
There'll obviously be more functions to fill in here for each of the sub-directories, but I like that theoretically, this orchestration skeleton lets us parallelize the work around fleshing out each of the functions that goes into making the pipeline run. As long as each of us is working on a different one of these sub-files, and as long as we document what the inputs/outputs to each one w.r.t. to the main pipeline are, then we should be able to refine all of the functions simultaneously.